### ISSUE TITLE
prune images of schema 2 - should prune old image with config

### ISSUE THREAD

**Issue by soltysh** [openshift/origin#12499]
I'm seeing this extended test failure both in current master and in the current rebase:

```
• Failure [56.993 seconds]
[images] prune images
/data/src/github.com/openshift/origin/test/extended/images/prune.go:85
  of schema 2
  /data/src/github.com/openshift/origin/test/extended/images/prune.go:84
    should prune old image with config [It]
    /data/src/github.com/openshift/origin/test/extended/images/prune.go:83

    Expected
        <string>: Dry run enabled - no modifications will be made. Add --confirm to remove images
        Deleting references from image streams to images ...
        STREAM                                         IMAGE                                                                     TAGS
        extended-test-prune-images-3lzp8-dvg65/prune   sha256:989d62758d6a0b8a0c326d2ed84e5b9eaa4975f8e9a7880f7b1b55a5f77930d1   latest
        
        Deleting registry repository layer links ...
        REPO                                           LAYER LINK
        extended-test-prune-images-3lzp8-dvg65/prune   sha256:f1041d65727d2d7ee29d8f3f8975d2782b34246b100579b978ffaa9338816b7a
        extended-test-prune-images-3lzp8-dvg65/prune   sha256:09ec8a0721608045992660137db1cc817b0fd945214b2452adad00bc5e12e7e8
        
        Deleting registry layer blobs ...
        BLOB
        sha256:f1041d65727d2d7ee29d8f3f8975d2782b34246b100579b978ffaa9338816b7a
        sha256:e7e405bfcd8e5ae1e1b162c1f0a8f51ccfd76e5b04f55f4cc0b4ade19ea920e8
        sha256:09ec8a0721608045992660137db1cc817b0fd945214b2452adad00bc5e12e7e8
        sha256:cbd820c371f7c529183be0b318fc8763769817b5a229f911d9aa5064ee80f59a
        sha256:729b1393f8be141b88c8d7e220f12fb24121feabe6dbe5d291b6a6cc02bbff7c
        sha256:6d5981658b9e8728da196504c0818138a86d96e8ccafefa5e01672800d1a0f1f
        sha256:01875e248676e4a572ebc1659cef0effa5939d3e11130fa514d6220ec2bada20
        sha256:1606620ef7ebe33d68dc819980b2ae7e0ad43a0c33ece9833a9fc468307286c3
        sha256:2a88dc48f0522f21ec6889dd6177ef1b6132cedf10cad95b461759cc66343a1c
        sha256:d6d897d19b25daf0149bb3b1f70f9eb3c0b35df82db95e8999a8802e7136630b
        sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
        sha256:45a2e645736c4c66ef34acce2407ded21f7a9b231199d3b92d6c9776df264729
        sha256:b74941a5e64da5fe7f91e8e3e40529661b18b3e54ecacabf7b82a920a606921e
        sha256:23f0135afcd024032f3bba55e857222dd5a4932758ca22e93caa3f10945e18eb
        sha256:f7d81f468b937c89cd7803f82eb0441988fd31ae90429f33a731d0b656cfe26a
        sha256:50d0864be036eb14696d1c889d7884b863e4bc65bb79ddfb3160f3727cd1845d
        sha256:3f92963ff97b7ecef976010b5e1d257ca91ecf8a6bc70542c6dc4540b587ed7b
        
        Deleting registry repository manifest data ...
        REPO                                           IMAGE
        extended-test-prune-images-3lzp8-dvg65/prune   sha256:989d62758d6a0b8a0c326d2ed84e5b9eaa4975f8e9a7880f7b1b55a5f77930d1
        
        Deleting images from server ...
        IMAGE
        sha256:5b4513a1d8cc9cec26539c85ec33a84fb888a88192180d38ebf12b83767e8875
        sha256:1ac92c08fe32224683a82556b96ae94ee5f67381bd53ef671b3e9b7869959746
        sha256:9a0e7809e9b5ad1fd706e7e66db51fdc5d6604123ffd28465e66445f8f2fad5d
        sha256:9fca5cd08b22948edb51b5238a2ffc5e63390809aa325b58a85bf4b525d19603
        sha256:077bf3c64737df592307720f9e342b76a46830fed0885cab58c229273c93ae30
        sha256:989d62758d6a0b8a0c326d2ed84e5b9eaa4975f8e9a7880f7b1b55a5f77930d1
    to contain substring
        <string>: sha256:49cbcaeec119cd52cc777fd0b62cb5ad44103eae1cba41d6c10be8c822fe3650

    /data/src/github.com/openshift/origin/test/extended/images/prune.go:136
```

**Comment by soltysh** [comment]
Seen in https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/984/consoleFull or generally on every run I do locally.

**Comment by miminar** [comment]
Fixed by #12566
